internal/cmd/load-balancer/target-pool/describe/describe.go

```diff
@@ -0,0 +1,189 @@
+}
+
+func configureFlags(cmd *cobra.Command) {
+	cmd.Flags().String(lbNameFlag, "", "Name of the load balancer")
+
+	err := flags.MarkFlagsRequired(cmd, lbNameFlag)
+	cobra.CheckErr(err)
+}
+
+func parseInput(p *print.Printer, cmd *cobra.Command, inputArgs []string) (*inputModel, error) {
+	targetPoolName := inputArgs[0]
+
+	globalFlags := globalflags.Parse(p, cmd)
+	if globalFlags.ProjectId == "" {
+		return nil, &errors.ProjectIdError{}
+	}
+
+	model := inputModel{
+		GlobalFlagModel: globalFlags,
+		TargetPoolName:  targetPoolName,
+		LBName:          cmd.Flag(lbNameFlag).Value.String(),
+	}
+
+	if p.IsVerbosityDebug() {
+		modelStr, err := print.BuildDebugStrFromInputModel(model)
+		if err != nil {
+			p.Debug(print.ErrorLevel, "convert model to string for debugging: %v", err)
+		} else {
+			p.Debug(print.DebugLevel, "parsed input values: %s", modelStr)
+		}
+	}
+
+	return &model, nil
+}
+
+func buildRequest(ctx context.Context, model *inputModel, apiClient *loadbalancer.APIClient) loadbalancer.ApiGetLoadBalancerRequest {
+	req := apiClient.GetLoadBalancer(ctx, model.ProjectId, model.LBName)
+	return req
+}
+
+func outputResult(p *print.Printer, model *inputModel, loadBalancer *loadbalancer.LoadBalancer) error {
+	switch model.OutputFormat {
+	case print.JSONOutputFormat:
+		details, err := json.MarshalIndent(loadBalancer, "", "  ")
+		if err != nil {
+			return fmt.Errorf("marshal load balancer: %w", err)
+		}
+		p.Outputln(string(details))
+
+		return nil
+	default:
+		return outputResultAsTable(p, model.TargetPoolName, loadBalancer)
+	}
+}
+
+func outputResultAsTable(p *print.Printer, targetPoolName string, loadBalancer *loadbalancer.LoadBalancer) error {
+	targetPool := utils.FindLoadBalancerTargetPoolByName(loadBalancer.TargetPools, targetPoolName)
+
+	sessionPersistence := "None"
+	if targetPool.SessionPersistence != nil && targetPool.SessionPersistence.UseSourceIpAddress != nil && *targetPool.SessionPersistence.UseSourceIpAddress {
+		sessionPersistence = "Use Source IP"
+	}
+
+	healthCheckInterval := "-"
+	healthCheckUnhealthyThreshold := "-"
+	healthCheckHealthyThreshold := "-"
+	healthCheckTimeout := "-"
+	if targetPool.ActiveHealthCheck != nil {
+		if targetPool.ActiveHealthCheck.Interval != nil {
+			healthCheckInterval = *targetPool.ActiveHealthCheck.Interval
+		}
+		if targetPool.ActiveHealthCheck.UnhealthyThreshold != nil {
+			healthCheckUnhealthyThreshold = strconv.FormatInt(*targetPool.ActiveHealthCheck.UnhealthyThreshold, 10)
+		}
+		if targetPool.ActiveHealthCheck.HealthyThreshold != nil {
+			healthCheckHealthyThreshold = strconv.FormatInt(*targetPool.ActiveHealthCheck.HealthyThreshold, 10)
+		}
+		if targetPool.ActiveHealthCheck.Timeout != nil {
+			healthCheckTimeout = *targetPool.ActiveHealthCheck.Timeout
+		}
+	}
+
+	targets := "-"
+	if targetPool.Targets != nil {
+		var targetsSlice []string
+		for _, target := range *targetPool.Targets {
+			targetStr := fmt.Sprintf("%s: %s", *target.DisplayName, *target.Ip)
```

Comment:
The Portal shows target as
```suggestion
			targetStr := fmt.Sprintf("%s (%s)", *target.DisplayName, *target.Ip)
```

I think that might be a bit clearer